R/render.R

```diff
@@ -296,7 +298,7 @@ use_lang_chapter <- function(chapters_list, language_code,
       language_code = language_code
     )
 
-    if (any(!fs::file_exists(chapters_list[["chapters"]]))) {
+    if (!all(fs::file_exists(chapters_list[["chapters"]]))) {
       chapters_not_translated <- !fs::file_exists(chapters_list[["chapters"]])
```

Comment:
why don't we have all/any here (my fault I know)?

Reply:
Because we aren't trying to "summarize" it to one boolean? This creates a vector of T/F used on L306 to get the missing chapters we copy from the mainlanguage.

Maybe I'm missing something but I don't see why we should use any() or all()?

Reply:
oh right :woman_facepalming: